dali/python/CMakeLists.txt

```diff
@@ -36,7 +36,49 @@ install(FILES "${PROJECT_SOURCE_DIR}/dali/python/setup.py" DESTINATION "${PROJEC
 install(FILES "${PROJECT_SOURCE_DIR}/tools/rec2idx.py" DESTINATION "${PROJECT_BINARY_DIR}/dali/python")
 install(FILES "${PROJECT_SOURCE_DIR}/tools/tfrecord2idx" DESTINATION "${PROJECT_BINARY_DIR}/dali/python")
 
+add_custom_command(
+    TARGET ${dali_python_lib} POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy
+            "${PROJECT_SOURCE_DIR}/dali/python/nvidia"
+            "${PROJECT_BINARY_DIR}/dali/python")
+
+add_custom_command(
+    TARGET ${dali_python_lib} POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy
+            "${PROJECT_SOURCE_DIR}/dali/python/setup.py"
+            "${PROJECT_BINARY_DIR}/dali/python")
+
+add_custom_command(
+    TARGET ${dali_python_lib} POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy
+            "${PROJECT_SOURCE_DIR}/tools/rec2idx.py"
+            "${PROJECT_BINARY_DIR}/dali/python")
+
+add_custom_command(
+    TARGET ${dali_python_lib} POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy
+            "${PROJECT_SOURCE_DIR}/tools/tfrecord2idx"
+            "${PROJECT_BINARY_DIR}/dali/python")
+
 # Get the COPYRIGHT, LICENSE, and Acknowledgements
 install(FILES "${PROJECT_SOURCE_DIR}/Acknowledgements.txt" DESTINATION "${PROJECT_BINARY_DIR}/dali/python/nvidia/dali")
 install(FILES "${PROJECT_SOURCE_DIR}/COPYRIGHT" DESTINATION "${PROJECT_BINARY_DIR}/dali/python/nvidia/dali")
 install(FILES "${PROJECT_SOURCE_DIR}/LICENSE" DESTINATION "${PROJECT_BINARY_DIR}/dali/python/nvidia/dali")
```

Comment:
Same comment as above about whether we still need these install lines

Reply:
Done